napari/utils/event.py

```diff
@@ -665,10 +674,11 @@ class EmitterGroup(EventEmitter):
         See the :func:`add <vispy.event.EmitterGroup.add>` method.
     """
 
-    def __init__(self, source=None, auto_connect=True, **emitters):
+    def __init__(self, source=None, event_handler_callback=None, **emitters):
```

Comment:
why removing `auto_connect` here?  (also note, that kwarg is still being used in `Layer.__init__`)

Reply:
Also concerned about this... Again, it would be best to make any changes be backwards compatible here.

Reply:
Yeah, this is a mystery to me right now, not all events connections are working as expected and I can't figure out why - specifically `layer.events.name.connect(lambda e: layers._update_name(e))` in https://github.com/shanaxel42/napari/pull/2 which I unfortunately still need